qa/workunits/cephtool/test.sh

```diff
@@ -575,9 +576,9 @@ function test_auth()
   check_response "auid = $auid"
   ceph --format json-pretty auth get client.TEST > $TMPFILE
   check_response '"auid": '$auid
-  ceph auth list > $TMPFILE
+  ceph auth ls > $TMPFILE
```

Comment:
@tchaikov Thanks, kefu.  I get confused because Sage directly updates the old format here and below.